device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,326 @@
+{
+    "chassis": {
+        "name": "MSN2700",
+        "components": [],
```

Comment:
mellanox provide the names of the components

Reply:
@keboliu: After reviewing the Mellanox platform API code ([here](https://github.com/Azure/sonic-buildimage/blob/master/platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py#L156) and [here](https://github.com/Azure/sonic-buildimage/blob/master/platform/mellanox/mlnx-platform-api/sonic_platform/component.py)), it appears as though the expected component list for this platform should be as follows. Can you please confirm?

```
"components": [],
    {
        "name": "ONIE"
    },
    {
        "name": "SSD"
    },
    {
        "name": "BIOS"
    },
    {
        "name": "CPLD1"
    },
    {
        "name": "CPLD2"
    },
    {
        "name": "CPLD3"
    }
],
```

Reply:
correct

Reply:
fixed